src/components/themed/WalletListHeader.tsx

```diff
@@ -48,12 +47,6 @@ export class WalletListHeaderComponent extends React.PureComponent<Props> {
             <SectionHeaderUi4 leftTitle={lstrings.title_wallets} rightNode={addSortButtons} />
           </EdgeAnim>
         )}
-
-        {searching ? null : (
-          <EdgeAnim enter={{ type: 'fadeInUp', distance: 20 }} exit={{ type: 'fadeOutUp' }}>
-            <PromoCard navigation={navigation} />
-          </EdgeAnim>
-        )}
```

Comment:
This isn't in the scope of this PR, so could you give me the task to understand what/why for this change?

Reply:
Also, provide screenshot if applicable